.github/workflows/test.yml

```diff
@@ -19,12 +19,10 @@ jobs:
           go-version: "1.21"
           check-latest: true
 
-      # TODO: uncomment after migrating to Go >= 1.16 or after go mod tidy
-      # Error: ../../../go/pkg/mod/github.com/daixiang0/gci@v0.3.3/pkg/io/search.go:4:2: package io/fs is not in GOROOT (/opt/hostedtoolcache/go/1.15.15/x64/src/io/fs)
-      # - name: Lint
-      #   run: |
-      #     go get github.com/golangci/golangci-lint/cmd/golangci-lint
-      #     make lint
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@23faadfdeb23a6f9e511beaba149bb123b5b145a # v6
+        with:
+          version: v1.58.0
```

Comment:
Just realised this could extracted in a separate workflow. The benefit is that it will run in parallel with other workflows.

Reply:
Not sure if you need this right now, just an option.

Reply:
Not insisting. Speed of execution is not the only concern. It also allows to separate running tests from the linter. For instance, the linter might fail for a style reason, but tests would still pass. If linter is preceding tests, then it's sequential, and a linter failure won't run tests.

But that's up to a project to decide. In bat-go, we run linters in parallel (as it takes a bit longer, and it also makes sense for the reason mentioned above).